bundles/org.openhab.binding.nuki/README.md

```diff
@@ -34,35 +34,146 @@ The following configuration options are available:
 | port            | The Port which you configured during [Initial Bridge setup](https://nuki.io/en/support/bridge/bridge-setup/initial-bridge-setup/).                                                                 | Default 8080 |
 | apiToken        | The API Token which you configured during [Initial Bridge setup](https://nuki.io/en/support/bridge/bridge-setup/initial-bridge-setup/).                                                            | Required     |
 | manageCallbacks | Let the Nuki Binding manage the callbacks on the Nuki Bridge. It will add the required callback on the Nuki Bridge. If there are already 3 callbacks, it **will delete** the callback with ID `0`. | Default true |
+| secureToken     | Whether hashed token should be used when communicating with Nuki Bridge. If disabled, API token will be sent in plaintext with each request.                                                       | Default true |
```

Comment:
Is there any reason sending it in plaintext? See my comment in the XML file.

Reply:
Yes, I also mentioned it in documentation. The hashed token also contains current timestamp to prevent replay attacks and only works if time on openhab and time on nuki bridge are synchronized (within a minute or so). If you can't keep openhab and nuki bridge time synchronized (for example you don't have internet connection and nuki bridge can't reach ntp server), then it stops working. I actually added this option after it happened to me when I was testing the binding.